code/modules/integrated_electronics/subtypes/output.dm

```diff
@@ -462,7 +462,7 @@
 
 
 /obj/item/integrated_circuit/output/holographic_projector/proc/destroy_hologram()
-	hologram.forceMove(src)
+	hologram = null
```

Comment:
Will this not cause the qdel on the next line to simply be attempting to delete a null object?

Reply:
Ah, you're right. It should be in the opposite order. I suppose I could've just used `qdel_null()`.